sdk/src/_composables/rest.ts

```diff
@@ -28,7 +28,7 @@ export interface RESTClient<Schema extends object> {
 	): Promise<Output>;
 }
 
-export function REST(cfg: RESTConfig = {}) {
+export function rest(cfg: RESTConfig = {}) {
```

Comment:
Opinion: I think I prefer keeping these lowercase. Very similar to express middleware